[2/3] [JIT] Make sure fusion occurs in test_tensorexpr

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #45791 [JIT] Bind log1p and lgamma
* #45790 [JIT] [3/3] Make sure fusion occurs in test_tensorexpr
* **#45789 [2/3] [JIT] Make sure fusion occurs in test_tensorexpr**
* #45788 [1/3] [JIT] Make sure fusion occurs in test_tensorexpr file

Making sure that more tests invoke a run with a Fusion Group.

Differential Revision: [D24169535](https://our.internmc.facebook.com/intern/diff/D24169535)

Commit:
Batch of fixes

[ghstack-poisoned]

Commit:
Update on "Batch of fixes"




[ghstack-poisoned]

Krovatkin reviewed (approved):
:shipit:

ZolotukhinM commented on torch/csrc/jit/runtime/graph_executor.cpp:
```diff
@@ -85,6 +85,17 @@ bool getAutodiffSubgraphInlining() {
 
+// for debugging it is helpful to be able to force fusion groups
+// to be created
+thread_local bool fusion_group_inlining = true;
```
Nit: could be made more configurable if it specifies min group size.

Commit:
Update on "[2/3] [JIT] Make sure fusion occurs in test_tensorexpr"


Making sure that more tests invoke a run with a Fusion Group.





[ghstack-poisoned]

Commit:
Update on "[2/3] [JIT] Make sure fusion occurs in test_tensorexpr"


Making sure that more tests invoke a run with a Fusion Group.

Differential Revision: [D24169535](https://our.internmc.facebook.com/intern/diff/D24169535)

[ghstack-poisoned]
